plugins/Monolog/Handler/WebNotificationHandler.php

```diff
@@ -58,4 +59,15 @@ protected function write(array $record)
             // Silently ignore the error.
         }
     }
+
+    private function getLiteDebuggingInfo()
+    {
+        $module = Common::getRequestVar('module', false);
```

Comment:
Might need to use an empty string here by default so something is visible also for the others? like `'""'` or even better not show the info if no value is present?

Reply:
I can remove it if no value is present.

Reply:
Be good 👍